components/OfferSearch.tsx

```diff
@@ -24,12 +24,18 @@ export type OfferSearchProps = {
 	offerType: Record<string, any>;
 };
 
+const getInitShowFilters = (): boolean => {
+	return window.innerWidth > 768; // tailwind css breakpoint
```

Comment:
maybe we can have a `const` for the breakpoint (in `utils/constants.ts` file)? I think we might need to use it soon to control other behavior...

Reply:
you're right, I considered it, but not this time, with second usage... ;)

it should be done via tailwind.config.js https://tailwindcss.com/docs/screens 
which can be imported into custom functions as a single source of truth